common/src/View/QtUtils.cpp

```diff
@@ -73,6 +71,31 @@ namespace TrenchBroom {
             m_widget->setUpdatesEnabled(true);
         }
 
+        /**
+         * Helper function to find an existing settings file, or build the path to such a file if it doesn't exist yet.
+         */
+        QString getSettingsFilePath();
```

Comment:
Remove extra function prototype, also make the function static since it's only used in this file

Reply:
method is removed